rasa_core/train.py

```diff
@@ -232,6 +233,9 @@ def train_dialogue_model(domain_file, stories_file, output_path,
     _interpreter = NaturalLanguageInterpreter.create(cmdline_args.nlu,
                                                      _endpoints.nlu)
 
+    domain = TemplateDomain.load(cmdline_args.domain)
+    _tracker_store = TrackerStore(domain).find_tracker_store(
```

Comment:
```
TrackerStore.find_tracker_store(domain, _endpoints.tracker_store)
```

Reply:
jeez i missed it :P sorry about that

Reply:
Absolutely no problem. Thanks for being so responsive and contributing!